src/main/kotlin/Book.kt

```diff
@@ -0,0 +1,12 @@
+class Book(override val wordCount: Int, override val price: Int) : Publication {
+    override fun getType() {
+        if (wordCount <= 1000) {
+            println("Flash Fiction")
+        } else if (wordCount <= 7500) {
+            println("Short Story")
+        } else println("Novel")
```

Comment:
Зачем else if?
Можно 
```suggestion
        when {
            wordCount <= 1000 -> println("Flash Fiction")
            wordCount <= 7500 -> println("Short Story")
            else -> println("Novel")
        }
 ```